src/store/mutations/mutations-model.ts

```diff
@@ -1,5 +1,17 @@
 import { CertifyStatementIF, CertifyIF, IncorporationAddressIf, NameRequestIF,
-  ExternalResourceIF, BusinessContactIF, OrgPersonIF, ShareClassIF } from '@/interfaces'
+  BusinessContactIF, OrgPersonIF, ShareClassIF } from '@/interfaces'
+
+export const mutateCertifyStatementResource = (state: any, certifyStatementResource: CertifyStatementIF) => {
+  state.resourceModel.certifyStatementResource = certifyStatementResource
+}
+
+export const mutateKeycloakRoles = (state: any, keyCloakRoles: Array<string>) => {
+  state.stateModel.tombstone.keycloakRoles = keyCloakRoles
+}
+
+export const mutateAuthRoles = (state: any, authRoles: Array<string>) => {
+  state.stateModel.tombstone.authRoles = authRoles
+}
```

Comment:
I moved these up from below to keep them together and apart from the rest (ie, tombstone vs other properties).